vipps-recurring-api.md

```diff
@@ -637,9 +740,13 @@ For example, in the agreement
 a limit of 2495 NOK (499 x 5) would be in place. If this limit becomes a
 hindrance the agreement `price` can be [updated](#update-an-agreement).
 
-**Note:** Although it is _technically_ possible to increase the price 10
-times, we **strongly** recommend to be as user-friendly as possible, and
-to make sure the user understands any changes and get updated information.
+From **August 1st, 2022** new rules will be enforced : 
```

Comment:
Shouldn't have a space before the ":". Not a biggie though. I can fix it later.

Reply:
Good catch, fixed it everywhere else in the file as well. 
It's a habit I have from French 🇫🇷 , we put a space before the ":" 😄